pkg/KubeArmorOperator/Dockerfile

```
@@ -37,7 +37,29 @@ COPY $OPERATOR_DIR/runtime runtime
 # Build
 RUN CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH} GO111MODULE=on go build -a -o operator cmd/main.go
 
-FROM scratch
+FROM redhat/ubi9-minimal
+
+ARG VERSION=latest
+
+LABEL name="kubearmor-operator" \
+      vendor="Accuknox" \
+      version=${VERSION} \
+      release=${VERSION} \
+      summary="kubearmor-operator container image based on redhat ubi" \
+      description="kubearmor-operator to deploy and manage KubeArmor"
+
+RUN microdnf -y update && \
+    microdnf -y install --nodocs --setopt=install_weak_deps=0 --setopt=keepcache=0 shadow-utils libcap && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum
```

Shouldn't the folder name be dnf?
```suggestion
    rm -rf /var/cache/dnf
```

---

microdnf keep cache in `/var/cache/yum`. we can also remove this as cache will be removed by `microdnf clean all`.